assets/index.css

```diff
@@ -234,25 +239,23 @@ main>section:not(:first-child) {
   max-width: 840px;
 }
 
-.about__header,
-.features__header,
-.links__header {
-  font-weight: 500;
-}
-
 @media screen and (prefers-color-scheme: dark) {
   .hero__content {
     color: var(--text-color);
   }
 
   .hero__content__icon {
-    filter: brightness(75%);
+    filter: brightness(90%);
```

Comment:
Just to check, will it be blinding to the user or not?

Reply:
Yes. Has a good contrast ratio without blinding users. Logo colors were being affected at 75%.